gluoncv/data/otb/tracking.py

```diff
@@ -0,0 +1,211 @@
+"""Visual Tracker Benchmark."""
+import json
+import os
+from tqdm import tqdm
+from mxnet.gluon.data import dataset
+
+class Video(object):
+    """
+    Abstract video class. get video class information for example imgs.
+    Args:
+        name: video name
+        root: dataset root
+        video_dir: video directory
+        init_rect: init rectangle
+        img_names: image names
+        gt_rect: groundtruth rectangle
+        attr: attribute of video
+    """
+    def __init__(self, name, root, video_dir, init_rect, img_names,
+                 gt_rect, attr, load_img=False):
+        self.name = name
+        self.video_dir = video_dir
+        self.init_rect = init_rect
+        self.gt_traj = gt_rect
+        self.attr = attr
+        self.pred_trajs = {}
+        self.img_names = [os.path.join(root, x) for x in img_names]
+        self.imgs = None
+        self.cv2 = try_import_cv2()
```

Comment:
You need to import it before you can use it, `from gluoncv.utils.filesystem import try_import_cv2`